packages/scripts/promoteRelease.js

```diff
@@ -0,0 +1,242 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * @format
+ * @ts-check
+ */
+
+const _ = require('lodash');
+const chalk = require('chalk');
+const child_process = require('child_process');
+const findUp = require('find-up');
+const fs = require('fs');
+const path = require('path');
+const simplegit = require('simple-git/promise');
+const util = require('util');
+const yaml = require('js-yaml');
+const yargs = require('yargs');
+
+const glob = util.promisify(require('glob').glob);
+
+(async () => {
+  const argv = collectArgs();
+  const git = simplegit();
+  const branchName = `${argv.rnVersion}-stable`;
+  const commitMessage = `Promote ${argv.rnVersion} to ${argv.release}`;
+
+  if (argv.release === 'preview') {
+    console.log(`Creating branch ${branchName}...`);
+    await git.checkoutBranch(branchName, 'HEAD');
+  }
+
+  console.log('Updating Beachball configuration...');
+  await updateBeachballConfigs(argv.release, argv.rnVersion);
+
+  console.log('Updating CI variables...');
+  await writeAdoVariables({npmTag: distTag(argv.release, argv.rnVersion)});
+
+  if (argv.release === 'preview') {
+    console.log('Updating root change script...');
+    await updatePackage('react-native-windows-repo', {
+      scripts: {change: `beachball change --branch ${branchName}`},
+    });
+
+    console.log('Updating package versions...');
+    await updatePackageVersions(`${argv.rnVersion}-preview.0`);
+  }
```

Comment:
For preview we should also update the RN dependency from nightly builds to the first RC build of the version.

Reply:
Actually it might be best to keep this part manual, since the first RC seems to sometimes have some stuff picked into it.